src/main/java/seedu/multitasky/logic/parser/AddCommandParser.java

```diff
@@ -41,6 +41,7 @@ public AddCommand parse(String args) throws ParseException {
                                                  PREFIX_TO, PREFIX_TAG);
         Calendar startDate = null;
         Calendar endDate = null;
+        Prefix datePrefix;
 
```

Comment:
Maybe write header comments explaining what these multiple if-else conditions are doing? it's not very clear just reading through and this method is non-trivial